lib/src/main/java/io/ably/lib/realtime/Presence.java

```diff
@@ -192,6 +192,34 @@ public void leave(CompletionListener listener) throws AblyException {
 
 	/**
 	 * Enter a specified client into this channel. The given client will be added to the
+	 * presence set and presence subscribers will see an empty presence message.
```

Comment:
This is right but the wording is confusing.  I would instead say:
`The given clientId will be added to the presence set with an empty member data payload.`